src/WpBookingsFrontUi.php

```diff
@@ -50,192 +51,169 @@ public function __construct(
-                    return new OutputTemplateHandler($c->get('eddbk_front_components_templates'), [
-                        'eddbkWizardTemplate'            => $c->get('eddbk_front_wizard_template'),
-                        'sessionSelectorTemplate'        => $c->get('eddbk_front_session_selector_template'),
-                        'wizardServiceStepTemplate'      => $c->get('eddbk_front_service_step_template'),
-                        'wizardSessionStepTemplate'      => $c->get('eddbk_front_session_step_template'),
-                        'wizardConfirmationStepTemplate' => $c->get('eddbk_front_confirmation_step_template'),
-                    ]);
-                },
-
-                'eddbk_wizard_assets_urls_map' => function (ContainerInterface $c) {
-                    $containerFactory = $this->_getContainerFactory();
-                    $assetsUrlsMap = require_once $c->get('bookings_front_ui/assets_urls_map_path');
-
-                    return $containerFactory->make([
-                        ContainerFactoryInterface::K_DATA => $assetsUrlsMap,
-                    ]);
-                },
-
-                'eddbk_wizard_assets' => function (ContainerInterface $c) {
-                    return $c->get('bookings_front_ui/assets');
-                },
-
-                /*
-                 * Handles assets enqueuing for application.
-                 *
-                 * @since [*next-version*]
-                 */
-                'eddbk_wizard_enqueue_assets_handler' => function (ContainerInterface $c) {
-                    return new AssetsEnqueueHandler($c->get('eddbk_wizard_assets_urls_map'), $c->get('eddbk_wizard_assets'));
-                },
-
-                /*
-                 * Handles state output for client application.
-                 *
-                 * @since [*next-version*]
-                 */
-                'eddbk_wizard_enqueue_app_state_handler' => function (ContainerInterface $c) {
-                    return new StateEnqueueHandler(
-                        $c->get('bookings_front_ui/application_selector'),
-                        $c->get('bookings_front_ui/api_endpoint_urls'),
-                        $c->get('bookings_front_ui/booking_data_map'),
-                        $c->get('bookings_front_ui/initial_booking_transition'),
-                        $c->get('bookings_front_ui/formats/datetime')
-                    );
-                },
-
-                /*
-                 * Handles for outputting main component of application.
-                 *
-                 * @since [*next-version*]
-                 */
-                'eddbk_wizard_main_component_handler' => function (ContainerInterface $c) {
-                    return new MainComponentHandler(
-                        $c->get('eddbk_front_app_holder_template')
-                    );
-                },
-            ]);
+        return $this->_setupContainer($this->_loadPhpConfigFile(RC_BOOKINGS_FRONT_UI_MODULE_CONFIG), [
```

Comment:
Not sure why you did this. The original problem was this: because the square bracket is on the same line as the call to `_loadPhpConfigFile()`, it looks like they are related. But they are not related; they are separate arguments to `_setupContainer()`.

Reply:
I clearly see that now this is separate arguments. For me, square bracket is not related to function call. It looks like just your subjective perception.

Reply:
@mecha, thoughts?

Reply:
Doesn't matter what _I_ think. Doesn't matter what _you_ think. What matters is what `composer csfix` thinks ;)

Reply:
But I will say that AFAIK, PSR-2 coding standards says that parameters that are broken up need to be on separate lines. We only break this rule for exception-creation methods, which tend to bloat code quite a bit.

Reply:
Found it. [Section 4.6. Method and Function Calls](https://www.php-fig.org/psr/psr-2/#46-method-and-function-calls).

Reply:
At the same time, https://www.php-fig.org/psr/psr-2/meta/#31---multi-line-arguments-09082013